rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly add getters and setters method for book and person by adding them to attr_accessor.

Reply:
Idem for this!

Reply:
I can't do it because it will create an error in the method book and person. You didn't read the requirements or what?

Reply:
I would like to apologize but this requirement has to be done. Please chexk with full time tse for more information

Reply:
are you serious?